### PR TITLE
Add list_of_speakers action set.

### PR DESCRIPTION
**Pull request by reiterl**
Another action set: list_of_speakers.
Not complete model.
Add some tests.

### CODE REVIEW

File: openslides_backend/models/list_of_speakers.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    Reverse fields:
+    TODO
+    """
+
+    collection = Collection("list_of_speakers")
+    verbose_name = "list_of_speakers"
+
+    id = fields.IdField(description="The id of this list of speakers.")
+    meeting_id = fields.RequiredForeignKeyField(
+        description="The id of the meeting of this motion workflow.",
```
Comment (jsangmeister) [inline openslides_backend/models/list_of_speakers.py:19]: -> "list of speakers"

File: openslides_backend/models/list_of_speakers.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from ..shared.patterns import Collection
+from . import fields
+from .base import Model
+
+
+class ListOfSpeakers(Model):
+    """
+    Model for motion workflow.
```
Comment (jsangmeister) [inline openslides_backend/models/list_of_speakers.py:8]: -> "list of speakers"

File: openslides_backend/models/list_of_speakers.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    collection = Collection("list_of_speakers")
+    verbose_name = "list_of_speakers"
+
+    id = fields.IdField(description="The id of this list of speakers.")
+    meeting_id = fields.RequiredForeignKeyField(
+        description="The id of the meeting of this motion workflow.",
+        to=Collection("meeting"),
+        related_name="list_of_speakers_ids",
+    )
+    # TODO closed: boolean;
```
Comment (jsangmeister) [inline openslides_backend/models/list_of_speakers.py:23]: see #101

File: openslides_backend/models/list_of_speakers.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/models/list_of_speakers.py]: Update description.

File: openslides_backend/models/list_of_speakers.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/models/list_of_speakers.py]: Update docstring.

File: openslides_backend/models/list_of_speakers.py (hunk 3 above)
Comment (reiterl) [reply openslides_backend/models/list_of_speakers.py]: Merged #101 and add the boolean field.

**Comment by jsangmeister** [comment]
See https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/9 and https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/11

**Review by jsangmeister — CHANGES_REQUESTED** [review]
See https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/9, https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/10 and https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/11. You cannot create/delete lists of speakers, but we need a helper function of some kind to create it from other actions. You can look into how to handle this the best. Also, the update action needs to be implemented. The action set can then be removed.

**Comment by reiterl** [comment]
Updated, rm create and delete actions and update the tests of the update action.

File: openslides_backend/models/list_of_speakers.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    verbose_name = "list_of_speakers"
+
+    id = fields.IdField(description="The id of this list of speakers.")
+    meeting_id = fields.RequiredForeignKeyField(
+        description="The id of the meeting of this list of speakers.",
+        to=Collection("meeting"),
+        related_name="list_of_speakers_ids",
+    )
+    closed = fields.BooleanField(description="If the list of speakers is closed.")
+    # TODO content_object_id: */list_of_speakers_id;
```
Comment (normanjaeckel) [inline openslides_backend/models/list_of_speakers.py:26]: This is just a reverse field.

**Review by normanjaeckel — APPROVED** [review]

**Review by jsangmeister — APPROVED** [review]

File: openslides_backend/models/list_of_speakers.py (hunk 4 above)
Comment (reiterl) [reply openslides_backend/models/list_of_speakers.py]: Add this to the reverse fields. Also expand the reverse fields with the model information.

**Pull request merged**